Resources/Locale/en-US/holopad/holopad.ftl

```diff
@@ -141,6 +147,8 @@ holopad-service-musician = Service - Musician
 holopad-service-mime = Service - Mime
 holopad-service-clown-mime = Service - Clown/Mime
 holopad-service-gameroom = Service - Game Room
+holopad-service-breakroom = Service - Breakroom
+holopad-service-toolroom = Service - Toolroom
```

Comment:
Isnt this just General - Tools?

Reply:
Again downstream. It's standard on our maps to have service locked tool rooms.